Sources/Document/Document.swift

```diff
@@ -81,8 +94,13 @@ public actor Document {
     private var presences: [ActorID: StringValueTypeDictionary]
 
     public init(key: String) {
```

Comment:
Probably better to declare as a convenience initializer

Reply:
The document is an actor. So I can't declare a convenience initializer.

<img width="488" alt="스크린샷 2023-09-25 오후 7 33 48" src="https://github.com/yorkie-team/yorkie-ios-sdk/assets/3667927/c2a1bfb2-afab-42d1-9d28-28a4a110fb6e">